<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeighScaleDialog</class>
 <widget class="QDialog" name="WeighScaleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weight Measurement</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="TestInfoWidget" name="testInfoWidget" native="true"/>
   </item>
   <item>
    <widget class="SerialPortWidget" name="serialPortPickerWidget" native="true"/>
   </item>
   <item>
    <widget class="MeasurementTable" name="measurementTable" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="optionsWidget">
     <property name="title">
      <string>Device Controls</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="zeroDevicePushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Zero Device</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MeasurementTable</class>
   <extends>QWidget</extends>
   <header>widgets/measurement_table.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TestInfoWidget</class>
   <extends>QWidget</extends>
   <header>widgets/test_info_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SerialPortWidget</class>
   <extends>QWidget</extends>
   <header>widgets/serial_port_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
